openwillis/measures/audio/util/transcribe_util.py

```diff
@@ -118,11 +118,12 @@ def get_clinical_labels(scale, measures, content_dict, json_response):
     """
     #Check if content is available for all the speaker
     if content_dict and content_dict['speaker0'] and content_dict['speaker1']:
-        if scale.lower() not in measures['scale'].strip("[]").replace(" ", "").split(","):
+        if scale.lower() not in measures['scale'].split(","):
             return json_response
-
-        spk1_score = sutil.match_transcript(measures, content_dict['speaker0'])
-        spk2_score = sutil.match_transcript(measures, content_dict['speaker1'])
+        
+        score_string = scale.lower()+'_string'
+        spk1_score = sutil.match_transcript(score_string, content_dict['speaker0'])
+        spk2_score = sutil.match_transcript(score_string, content_dict['speaker1'])
```

Comment:
Aren't we supposed to input here the transcript of the script, i.e. `measures[score_string]`, instead of `score_string`

Reply:
score_string = scale.lower()+'_string'

in score_string, we are handling dynamic score values (panss/madrs/updrs etc.), this is going to be user input, we can not hard code them.

If user pass score as "panss" then score_string is = "panss_string", which is defined in the config file with the same name.